aha-table.html

```diff
@@ -179,6 +179,12 @@
         type: Boolean,
         value: false
       },
+      //greedy: if table needs to be greedy
```

Comment:
Here are examples of comments for properties just above: 
//tableRows: if table row is striped
//tableColumns: if table row is striped
//filterable: if table row is filterable
//sortedColumn: sorted column name

Continuing this pattern, I will change my description to : 
//greedy-height: if table has a greedy height